src/lib/admin/QuestionGeneration.svelte

```
@@ -228,37 +243,50 @@
 							<div class="flex items-start gap-3">
 								<CheckCircle class="text-success mt-0.5 shrink-0" size={16} />
-								<span class="text-xs"><strong>Custom Tailored:</strong> Questions specific to your exact curriculum.</span>
+								<span class="text-xs"
+									><strong>Custom Tailored:</strong> Questions specific to your exact curriculum.</span
+								>
 							</div>
 						</div>
 						<div class="flex flex-col gap-3 w-full">
-							<a href="/sign-up" target="_blank" class="btn btn-primary btn-block">Upgrade to Pro</a>
-							<button class="btn btn-ghost btn-xs" onclick={() => limitReached = false}>Maybe later</button>
+							<a href="/sign-up" target="_blank" class="btn btn-primary btn-block">Upgrade to Pro</a
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)